Makefile

```diff
@@ -2,8 +2,8 @@ test:
 	REUSE_DB=1 ./manage.py test
 
 setup:
-	pip install -r requirements.txt
-	pip install django-nose
+	# pip install -r requirements.txt
+	# pip install django-nose
```

Comment:
I'm pretty sure it was because I made the makefile executable. my bad :crying_cat_face: